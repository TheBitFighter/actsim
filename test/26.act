defproc test () {

	int i;

	int ar[2];

	chp {
		i := 0;
		*[ i < 2 -> ar[i] := 0; i := i + 1]
		;log(ar[0]), log(ar[1])
	//	;i := 0;
	//	 log(ar[i]), log(ar[i+1])
	}

}
