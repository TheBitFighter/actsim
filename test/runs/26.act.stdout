[      50 t#:0] <> 0
[      50 t#:1] <> 0
